examples/comet/comet-pytorch.ipynb

```diff
@@ -0,0 +1,439 @@
+    "\n",
+    "The core class of Comet is an Experiment. An Experiment allows you to log your model hyperparameters, code, metrics and model configuration. The `Experiment` class pulls the credentials you set earlier to properly authenticate your session. After you run this line, you will see a link to comet.ml where you can view the modeling logs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed94a8aa-6017-4a3a-8512-cb4f00b80a41",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-04-19T22:38:43.412059Z",
+     "iopub.status.busy": "2022-04-19T22:38:43.411789Z",
+     "iopub.status.idle": "2022-04-19T22:38:44.936679Z",
+     "shell.execute_reply": "2022-04-19T22:38:44.936045Z",
+     "shell.execute_reply.started": "2022-04-19T22:38:43.412035Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "experiment = comet_ml.Experiment(project_name=\"pytorch\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e036a5b",
+   "metadata": {},
+   "source": [
+    "Next, we specify and log the hyperparameters for this session as a dictionary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a38f1991-b825-47ea-9fd7-e991726ee00b",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-04-19T22:38:49.589245Z",
+     "iopub.status.busy": "2022-04-19T22:38:49.587716Z",
+     "iopub.status.idle": "2022-04-19T22:38:49.598377Z",
+     "shell.execute_reply": "2022-04-19T22:38:49.597749Z",
+     "shell.execute_reply.started": "2022-04-19T22:38:49.589199Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "hyper_params = {\n",
+    "    \"sequence_length\": 28,\n",
+    "    \"input_size\": 28,\n",
+    "    \"hidden_size\": 128,\n",
+    "    \"num_layers\": 2,\n",
+    "    \"num_classes\": 10,\n",
+    "    \"batch_size\": 100,\n",
+    "    \"num_epochs\": 2,\n",
+    "    \"learning_rate\": 0.01,\n",
+    "}\n",
+    "experiment.log_parameters(hyper_params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c6ae141",
+   "metadata": {},
+   "source": [
+    "### Set up the modeling run\n",
+    "\n",
+    "The next cells are exactly the same as you would expect in any other PyTorch workflow. Load the MNIST dataset from `torchvison`, specify your dataloaders, define the model based on your hyperparameters, and define your loss and optimizer functions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9bba0f65-2d75-4498-903b-e10226e6c38d",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-04-19T22:38:49.912314Z",
+     "iopub.status.busy": "2022-04-19T22:38:49.912052Z",
+     "iopub.status.idle": "2022-04-19T22:38:50.111151Z",
+     "shell.execute_reply": "2022-04-19T22:38:50.110490Z",
+     "shell.execute_reply.started": "2022-04-19T22:38:49.912286Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# MNIST Dataset\n",
```

Comment:
Rather than having comments at the tops of cells I think you should have markdown between the cells. Or more broadly we should have the general style rule in our examples that no two code cells should be touching